project/SparkBuild.scala

```diff
@@ -53,7 +53,7 @@ object BuildCommons {
     core, graphx, mllib, mllibLocal, repl, networkCommon, networkShuffle, launcher, unsafe, testTags, sketch, _*
   ) = Seq(
```

Comment:
Can you change `testTags` in line 53 to `tags` and also update the corresponding code? Thanks.